data/blood/glucose/glucose.go

```diff
@@ -60,3 +61,49 @@ func NormalizeValueForUnits(value *float64, units *string) *float64 {
 	}
 	return value
 }
+
+const (
+	// MgdL_Per_MmolL renames MmolLToMgdLConversionFactor in an alternate way.
+	MgdL_Per_MmolL = MmolLToMgdLConversionFactor
+	// MmolL_Per_MgdL inverts the MmolL_Per_MgdL conversion factor.
+	MmolL_Per_MgdL = 1 / MgdL_Per_MmolL
+)
+
+// Convert between common blood glucose measurement units.
+//
+// Results are rounded to five decimal places. However, if fromUnits and
+// toUnits are the same, the value is returned unchanged. If either fromUnits
+// or toUnits aren't handled, Convert panics.
+//
+// Convert is modeled to return results as similar as possible to
+// NormalizeValueForUnits, but without the hassle of pointer values, and with
+// the ability to convert between arbitrary units (as supported).
+func Convert(value float64, fromUnits, toUnits string) float64 {
```

Comment:
[nit] If the function has multiple different types, then I prefer to be explicit about argument for make it easier to determine the type for each parameter (without having to use extra brain power to say "wait, there is no type specified here, okay, then keep reading until we find a type"). I know, extreme nit. :)